### PR TITLE
Start with audio/video muted

### PR DESCRIPTION
**Pull request by paweldomas**
Required for https://github.com/jitsi/jitsi-meet/pull/1819

### CODE REVIEW

File: modules/RTC/JitsiRemoteTrack.js [hunk 1]
```diff
@@ -214,11 +214,18 @@ export default class JitsiRemoteTrack extends JitsiTrack {
...
 
+        // The conference can be started without calling GUM
+        // FIXME if there would be a module for connection times this kind
+        // of logic (gumDuration or ttfm) should end up there
+        const gumStart = window.connectionTimes['obtainPermissions.start'];
+        const gumEnd = window.connectionTimes['obtainPermissions.end'];
+        const gumDuration
+            = !isNaN(gumEnd) && !isNaN(gumStart) ? gumEnd - gumStart : 0;
+
         const ttfm = now
```
Comment (bbaldino) [inline modules/RTC/JitsiRemoteTrack.js]: i know this was existing, but what does `ttfm` stand for?  could it have a more descriptive name?

Comment (paweldomas) [inline modules/RTC/JitsiRemoteTrack.js]: time to first media

File: modules/RTC/RTCUtils.js [hunk 2]
```diff
@@ -811,17 +764,9 @@ class RTCUtils extends Listenable {
                 const getUserMedia
                     = navigator.webkitGetUserMedia.bind(navigator);
 
-                if (navigator.mediaDevices) {
-                    this.getUserMedia = wrapGetUserMedia(getUserMedia);
-                    this.enumerateDevices
-                        = wrapEnumerateDevices(
-                            navigator.mediaDevices.enumerateDevices.bind(
-                                navigator.mediaDevices));
-                } else {
```
Comment (hristoterezov) [inline modules/RTC/RTCUtils.js]: We are not supporting anymore browsers where GUM is with callbacks or this is removed by mistake?

File: modules/RTC/RTCUtils.js [hunk 3]
```diff
@@ -888,10 +833,7 @@ class RTCUtils extends Listenable {
                         navigator.mediaDevices.getUserMedia.bind(
                             navigator.mediaDevices),
                             true);
-                this.enumerateDevices
-                    = wrapEnumerateDevices(
-                        navigator.mediaDevices.enumerateDevices.bind(
-                            navigator.mediaDevices));
+                this.enumerateDevices = rawEnumerateDevicesWithCallback;
```
Comment (hristoterezov) [inline modules/RTC/RTCUtils.js]: `rawEnumerateDevicesWithCallback` can be `undefined`.  Should we handle this somehow?

The old code will fail too if enumerateDevices is not defined. So this is not a regression for sure. I've just noticed it and decided to ask.

File: modules/RTC/RTCUtils.js (hunk 2 above)
Comment (paweldomas) [inline modules/RTC/RTCUtils.js]: That's what wrapGetUserMedia is doing isn't it ?

File: modules/RTC/RTCUtils.js (hunk 3 above)
Comment (paweldomas) [inline modules/RTC/RTCUtils.js]: It can be undefined but that's not what this PR is about. It was the same before, unless I'm missing something ?

**Pull request merged**